sailthru/sailthru_content/services/catalog_api_service.py

```diff
@@ -44,10 +45,45 @@ def _get_resource_from_api(self, api_endpoint, page_size, **kwargs):
 
         return results
 
+    def _get_resource_from_search_api(self, api_endpoint, **kwargs):
+        page = 1
+        response = api_endpoint.get(**kwargs)
```

Comment:
These statements are repeated with the statements in the while loop. I think you can find a way to set `next_page` to something that's truthful in the beginning and remove the repeated statements